backend/src/main/factories/application/controllers/rating/create-rating-controller-factory.ts

```diff
@@ -1,6 +1,12 @@
 import { Controller, CreateRatingController } from '@/application/controllers'
 import { makeCreateRatingValidation } from './create-rating-validation-factory'
+import { UserRepository } from '@/infra/repositories/mysql/user-repository'
+import { RatingRepository } from '@/infra/repositories/mysql/rating-repository'
+import { TicketRepository } from '@/infra/repositories/mysql/ticket-repository'
```

Comment:
Aqui dá pra fazer um refactor da seguinte forma:

Cria uma pasta repositories em `infra/repositories/mysql/` e move todos os `repository` para dentro dela.

Cria um `index.ts` na pasta mysql com o seguinte:

```typescript
export * from './repositores/rating-repository'
export * from './repositories/ticket-repository'
export * from './repositories/user-repository'

```

Com isso é possível alterar esses imports para:

```suggestion
import { UserRepository, RatingRepository, TicketRepository } from '@/infra/repositories/mysql'
```